README.md

```diff
@@ -2,3 +2,5 @@ ansible-cobbler
 ===============
 
 A simple Ansible role to install and configure Cobbler.
+
+To configure the cobbler role, adjust the settings in defaults/main.yml.
```

Comment:
The real intent is that you pull this role into a larger project and use the standard host and group variables. The defaults are setup primarily for testing in a Vagrant environment.